dusseldorf-ktor-unleash/pom.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>dusseldorf-ktor</artifactId>
+        <groupId>no.nav.helse</groupId>
+        <version>PLACEHOLDER</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>dusseldorf-ktor-unleash</artifactId>
+    <name>dusseldorf-ktor-unleash</name>
+
+    <properties>
+        <maven.compiler.source>11</maven.compiler.source>
+        <maven.compiler.target>11</maven.compiler.target>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>no.nav.helse</groupId>
+            <artifactId>dusseldorf-ktor-core</artifactId>
+            <version>${project.version}</version>
+            <scope>provided</scope>
+        </dependency>
+
+        <!-- https://mvnrepository.com/artifact/org.apache.logging.log4j/log4j-core -->
+        <dependency>
+            <groupId>org.apache.logging.log4j</groupId>
+            <artifactId>log4j-core</artifactId>
+            <version>2.14.1</version>
+        </dependency>
+
+
+        <!-- https://mvnrepository.com/artifact/com.silvercar.unleash-client-kotlin/unleash-client-kotlin -->
+        <dependency>
```

Comment:
Ta heller inn den "offisielle" modulen herfra https://github.com/Unleash/unleash-client-java 

Ser ut som denne (unleash-client-kotlin) kun er laget pga `Java 8 functionality that Android 21+ doesn't support removed.` som ikke er noe issue for oss, og den er ikke vedlikeholdt i noen særlig grad?

Reply:
Aha det la jeg ikke merke til. Will fix :)

Reply:
@fraadsbrandth se siste commit.